setup.py

```diff
@@ -57,6 +67,26 @@ def is_requirement(line):
     )
 
 
+def get_edx_platform_version():
+    """
+    Returns the pip package version for Open-edX
+    """
+    file_name = 'open_edx_version.txt'
+    command = 'pip show Open-edX > {}'.format(file_name)
+    subprocess.call(command, shell=True)
```

Comment:
Please, add a try-catch for CalledProcessError or another exception in order to stop the process if there is any problem. @andrey-canon

Reply:
Break line here. @andrey-canon

Reply:
is your suggestion catch the exception to avoid to raise it and then stop the process raising another exception ???

Reply:
Well, sorry, i didn't explain well, we need to catch the exception in order to log (not stop it) the error, and let the process move forward. @andrey-canon

Reply:
if the default case is an error, it's better raise the exception to show to the final user what it's  exactly happening

Reply:
Stop all the deploy process just to realize that we are using the incorrect version, if you log the error and don't stop the process, you can show to the user what it's exactly happening. @andrey-canon

Reply:
Why don't stop the process when the default case stops the process ???? suppress an exception and then raise another with less information

Reply:
Anywise, stop the process but, Why lees information? with the right information... RuntimeError("Unable to get the edX platform version while running {command}"), that's all.